ui/app/rif/actions.js

```diff
@@ -1,7 +1,12 @@
 const rifActions = {
```

Comment:
You have some conflicts here

Reply:
Actually thats not updated, maybe github its not showing the latest